userland_generic.ld

```diff
@@ -103,6 +103,15 @@ SECTIONS {
      */
     .stack :
     {
+        /* elf2tab requires that the `_SRAM_ORIGIN` symbol be present to
+		 * mark the first address in the SRAM memory. Since ELF files do
+		 * not really need to specify this address as they only care about
+		 * loading into flash, we need to manually mark this address for
+		 * elf2tab. elf2tab will use it to add a fixed address header in the
+		 * TBF header if needed.
+		 */
+	    _SRAM_ORIGIN = .;
```

Comment:
```suggestion
	    _sram_origin = .;
```

I don't necessarily feel strongly about this, but all of the ALL_CAPS variables in this file are used by the linker to control its operations, and all of the external symbols are `_underscore_lowercase`; better to match?

Reply:
Ha good point. I don't know why I was thinking caps. Probably because `ORIGIN(SRAM)` is all caps above.